docs/upgrade-guides/core/3-20-to-3-21.mdx

```diff
@@ -291,3 +291,17 @@ The `external-shipping-method-id` is no longer stored within `checkout.privateMe
 Previously, the `checkoutDeliveryMethodUpdate` and `checkoutShippingMethodUpdate` mutations would trigger the `CHECKOUT_UPDATED` webhook and invalidate taxes regardless of whether the same delivery method was already assigned to the checkout.
 Starting from version 3.21, if the same method that is already assigned to the checkout is provided, the `CHECKOUT_UPDATED` webhook will not be triggered, and taxes will not be invalidated.
 From Saleor's perspective, nothing has changed, as the checkout remains in the same state as it was before the mutation was called.
+
+
+## Introduced price freeze period for editable orders
+In previous versions, all editable orders (those with `DRAFT` and `UNCONFIRMED` status) refreshed their prices by retrieving them from current channel listings during each recalculation. 
```

Comment:
I would consider re-naming the `channel-listing` words to something more meaningful and not related to the implementation or API. Something like: `price for channel` or `actual price for channel`